tests/foreman/virtwho/utils.py

```diff
@@ -142,6 +145,59 @@ def get_virtwho_status():
         return 'undefined'
 
 
+def get_configure_id(name):
+    """Return the configure id by hammer.
+    :param str name: the configure name you have created.
+    :raises: VirtWhoError: If failed to get the configure info by hammer.
+    """
+    config = VirtWhoConfig.info({'name': name})
+    if 'id' in config['general-information']:
+        return config['general-information']['id']
+    else:
+        raise VirtWhoError("No configure id found for {}".format(name))
+
+
+def get_configure_command(config_id, org=1):
```

Comment:
'Default_Organization' is used for virt-who testing because the subscription manifest is belongs to the default org when we deployed satellite. Maybe we should get the org_id by the org_name(Default_Organization)

Reply:
Yes , you can get the ID by using org_name .